src/jvmMain/kotlin/io/github/oshai/kotlinlogging/logback/internal/LogbackLogEvent.kt

```diff
@@ -0,0 +1,48 @@
+package io.github.oshai.kotlinlogging.logback.internal
+
+import ch.qos.logback.classic.Logger
+import ch.qos.logback.classic.spi.LoggingEvent
+import io.github.oshai.kotlinlogging.KLoggingEventBuilder
+import io.github.oshai.kotlinlogging.Level
+import io.github.oshai.kotlinlogging.logback.toLogbackLevel
+
+public class LogbackLogEvent(
+  fqcn: String,
+  logger: Logger,
+  level: Level,
+  private val kLoggingEvent: KLoggingEventBuilder,
+) :
+  LoggingEvent(
+    fqcn,
+    logger,
+    level.toLogbackLevel(),
+    kLoggingEvent.internalCompilerData?.messageTemplate ?: kLoggingEvent.message,
```

Comment:
if you provide here message template, don't you need to also provide arguments?

Reply:
For my use case, I only need the template. And as the final log message has already been generated then there is no actual need for the arguments. So I would skip the arguments until someone comes and provides a real-world use-case where arguments are needed.

Reply:
Isn’t slf4j such a case? In case we want to support it later?

Reply:
Well, if we eagerly merge message template with the arguments then SLF4J has no need for the arguments -- the message is final and no further processing is needed.